CoastalResilienceNetwork/GeositeFramework
# Add new launchpad

## Overview

This PR adds a new, redesigned launchpad based on the design at https://marvelapp.com/19e970f/screen/16634903. The old launchpad is no longer need and was removed.

### Demo

![image](https://cloud.githubusercontent.com/assets/1042475/20542032/c4133ffa-b0cd-11e6-9c21-0e115c6c3284.png)

### Notes

Before getting started on this, I wasn't sure whether to make this a plugin or a framework component. In the end, I thought it would be less complicated to make it a plugin. However, implementing the follow-up issues may introduce some complexity when dealing with the Plugin API.

-  Activate the launchpad by default https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/747
- Finish styling https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/748
- Implement "view more" for plugin/scenario description in launchpad https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/749
- Shutdown launchpad when another plugin or subregion is activated https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/750

## Testing Instructions

- Start the app, and activate the launchpad from the sidebar.
- Click the buttons for activating the plugin and scenarios, and ensure that they activate/load.

Connects #736
